content/20.treatments.md

```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
+Standard of Care administered to all patients is defined by each study within the manuscript.
+Site(s) indicates the number and location(s) of the hospitals included in the study, except in the case where the analysis included all cases within a geographic area (Qatar, which had only 25 confirmed COVID-19 cases treated with TCZ in the observation period).
+The observation period refers to the timeperiod examined retrospectively to identify eligible patients.
+Primary outcomes, when specified, are indicated; clinical response is used to indicate studies that did not explicitly define a primary outcome and mainly reported changes in biomarkers, secondary infections, etc.
+Matched control refers to studies that sought to retrospectively identify pairs of comparable patients who received and didn't receive TCZ.
+PCR-confirmed indicates whether patients' COVID-19 diagnosis was confirmed by a laboratory test (RT-PCR).
+IMV = Invasive Medical Ventilation {#tbl:tcz-table}
+-->
+Many studies use a retrospective, observational design, where they compare outcomes for COVID-19 patients who received TCZ to those who did not over a set period of time.
+For example, one of the largest retrospective, observational analysis released to date [@doi:10/d2pk] compared the rates at which patients who received TCZ deceased or progressed to invasive medical ventilation over a 14-day period compared to patients receiving only SOC.
+
+Under this definition, SOC could include other drugs such as HCQ, azithromycin, lopinavir-ritonavir or darunavir-cobicistat, or heparin.
+While this study was not randomized, a subset of patients who were eligible to receive TCZ were unable to obtain it due to shortages; however, these groups were not directly compared in the analysis.
+After adjusting for variables such as age, sex, and SOFA score, they found that patients treated with TCZ were less likely to progress to invasive medical ventilation and/or death (adjusted hazard ratio = 0.61, CI 0.40-0.92, p=0.020), although analysis of death and ventilation separately suggests that this effect may have been driven by differences in the death rate (20% of control versus 7% of TCZ-treated patients).
+They reported particular benefits for patients whose P/F Ratio fell below a 150 mm Hg threshold and found no differences between groups administered subcutaneous versus intravenous TCZ.
+Another retrospective observational analysis of interest examined the charts of patients at a hospital in Connecticut, USA where 64% of all 239 COVID-19 patients in the study period were administered TCZ based on assignment by a standardized algorithm [@doi:10.1016/j.chest.2020.06.006].
+They found that survivorship was more similar in patients presenting with severe vs. nonsevere COVID-19  at intake (severe vs. nonsevere) when TCZ was administered and proposed that their algorithm was able to target TCZ to patients presenting with CRS.
```

Comment:
A little confused by this sentence. So patients with severe or nonsevere covid had similar survival when treated with TCZ?

Reply:
```suggestion
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, and they therefore proposed that their algorithm was able to identify patients presenting with CRS as good candidates for TCZ.
```
That is what they reported -- in the patients who got TCZ, their severity at intake was less strongly associated with whether they passed away.

Reply:
Just to clarify, "severe patients" are referring to those with CRS?

Reply:
It doesn't actually! Maybe this is better:
```suggestion
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, defined based on the amount of supplemental oxygen needed.
They therefore proposed that their algorithm was able to identify patients presenting with or likely to develop CRS as good candidates for TCZ.
```